docs/book/src/topics/federated-identity-credential.md

```diff
@@ -60,19 +59,19 @@ cat <<EOF > body.json
 }
 EOF
 
-az rest --method POST --uri "https://graph.microsoft.com/beta/applications/${APPLICATION_OBJECT_ID}/federatedIdentityCredentials" --body @body.json
+az ad app federated-credential create --id $APPLICATION_OBJECT_ID --parameters params.json
 ```
 
 To delete a federated identity credential, the federated identity credential ID needs to be obtained with the following command:
 
 ```bash
-az rest --method GET --uri "https://graph.microsoft.com/beta/applications/${APPLICATION_OBJECT_ID}/federatedIdentityCredentials"
+export FIC_ID="$(az ad app federated-credential list --id "${APPLICATION_OBJECT_ID}")"
```

Comment:
```suggestion
az ad app federated-credential list --id "${APPLICATION_OBJECT_ID}"
```

list returns an array of federated credential, so we should let the users pick the one they want to delete